governance/working groups/working-group-charter.md

```diff
@@ -0,0 +1,41 @@
+# Working group charter template
+
+## Overview
+
+This section should summarise in one or two paragraphs the Working group proposal, for any community member to read and quickly grasp the essence of the WG.
+
+## Background
+
+This section should dive into more detail on what the WG aims to address, why it is undertaking this work, and reference any background material/pre-existing work that this proposal builds on.
+
+## Target outputs
+This section should contain the target outputs of the WG
+
+## Meeting mechanisms
+This section should include details on how, where and when the WG intends to meet
+
+## Communication mechanisms
+This section should outline how the WG intends to communicate progress with its members, the UK TRE Community more widely, and any other interested stakeholders.
+
+## Working Group roles
+This section should outline the different roles within the Working Group. Examples include:
+- **Chairs**: Leaders of the WG
+- **Contact**: Primary contact for the WG
+- **Participating members**: Members who are undertaking work within the Working Group
```

Comment:
We could extend this out to e.g. informed members, experts etc but maybe overkill. [Inspiration](https://www.w3.org/2017/Process-20170301/#GAGeneral:~:text=There%20are%20three%20types%20of%20individual%20participants%20in%20a%20Working%20Group%3A%20Member%20representatives%2C%20Invited%20Experts%2C%20and%20Team%20representatives%20(including%20the%20Team%20Contact).

Reply:
I see where you're going, but I think we should keep it simple. I think my earlier comment about listing notable stakeholders can cover this point.